ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/ChooseConnectionSteps.java

```diff
@@ -26,7 +32,12 @@
     @Then("^check that position of connection to fill is \"([^\"]*)\"$")
     public void verifyTypeOfConnection(String position) {
         log.info("{} connection must be active", position);
-        assertTrue("There was no active icon found for position " + position, integrationFlowView.verifyActivePosition(position));
+        try {
+            OpenShiftWaitUtils.waitFor(() -> integrationFlowView.verifyActivePosition(position),
```

Comment:
Maybe use `TestUtils.waitFor(...)` method?

Reply:
Yup, you are right, it's more appropriate. Will fix in a few minutes.